test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -244,6 +250,65 @@
                             "found for service foo")}
               (json/parse-string (slurp (:body resp)))))))))
 
+
+(deftest simple-routes-params-ignoring-test
+  (with-status-service app
+    [foo-service]
+    (testing "ignores bad level"
+      (let [resp (http-client/get "http://localhost:8180/status/v1/simple?level=bar")]
+        (is (= 200 (:status resp)))
+        (is (= "running" (slurp (:body resp))))))
+    (testing "ignores alphabetic service_status_version"
+      (let [resp (http-client/get (str "http://localhost:8180/status/v1/"
+                                    "simple/foo?service_status_version=abc"))]
+        (is (= 200 (:status resp)))
+        (is (= "running" (slurp (:body resp))))))
+    (testing "ignores non-existent service_status_version"
+      (let [resp (http-client/get (str "http://localhost:8180/status/v1/"
+                                    "simple/foo?service_status_version=3"))]
```

Comment:
This indentation style isn't consistent with the rest of your PR (and also above on line 263).